dist_tables/ddl.rst

```diff
@@ -61,6 +61,30 @@ Each created shard is assigned a unique shard id and all its replicas have the s
 
 You are now ready to insert data into the distributed table and run queries on it. You can also learn more about the UDF used in this section in the :ref:`user_defined_functions` of our documentation.
 
+.. _colocation_groups:
+
+Co-Location Groups
+------------------
+
+Table rows are stored in shards across a Citus cluster. When all rows needed by a query are available in a single shard then the tables named by the query are said to be *co-located for the query*. (Tables which are co-located for the majority of queries in an application are simply called co-located tables.) Queries with co-located tables excute faster because Citus doesn't need to shuffle data over the network.
```

Comment:
One thing I was hoping to communicate is that co-location is something that holds between tables but only *with respect to certain queries.* Say you have two tables, `products` and `warrantees`, both distributed on a `manufacturer` column. Imagine the products are also sold to consumers at retail stores. The tables would be co-located for queries joining/grouping on manufacturer, but not co-located for queries like "per store, how many products are sold under warrantee?" So even though both tables are distributed by a column of the same type, can we categorically call them "co-located?"